tests/tests/test_keepalive.py

```diff
@@ -192,9 +192,14 @@ def test_keepalive_disable(self, monitor_commercial_setup_no_client):
         )
         assert get_opened_tcp_connections(mender_device) > 0
 
-        configure_connectivity(mender_device, disable_keep_alive=disable_keep_alive)
+        configure_connectivity(
+            mender_device,
+            disable_keep_alive=disable_keep_alive,
+            inventory_poll=3600,
+            update_poll=3600,
+        )
         logger.info("test_keepalive_disable: waiting for client to restart")
-        time.sleep(1)
+        time.sleep(15)
```

Comment:
no magic numbers please :) could we define variables for them?

Reply:
I could define a variable, but it will still be an arbitrary number which I think it just hides away the problem (which IMO is worst than being explicit and honest about it).

A different approach however is to retry the check until it passes or we reach a timeout. This we could do but I wanted to respect the original intention of the test in that regard.

Reply:
I think if we know that this is a hiding a problem and know how to rework it in a better way, then it's the way to go. This is agile approach and an original implementer should not be hurt by that.